File: Easy/Python/2000. Reverse Prefix of Word.py
class Solution(object):
    def reversePrefix(self, word, ch):
        wordChars = []

        if ch in word:
            for c in word:
                wordChars.append(c)
                if c == ch:
                    break
            
            wordChars.reverse()
            word = ''.join(wordChars) + word[word.index(ch) + 1:]

        return word

File: Easy/Java/2000. Reverse Prefix of Word.java
class Solution {
    public String reversePrefix(String word, char ch) {
        char[] wordChars = word.toCharArray();

        for(int i = 0; i <= word.indexOf(ch); i++){
            wordChars[i] = word.charAt(word.indexOf(ch) - i);
        }

        return String.valueOf(wordChars);
    }
}
